dev-resources/less/grid.less

```diff
@@ -317,7 +317,12 @@ border: 1px solid #afafaf background-header: #c8c8c8 background-odd: #fafafa bac
   padding: 1px 4px;
   z-index: 1;
 }
-
+.arvo-ja-nappi > * {
```

Comment:
Tiemerkintävalmiuden perumisessa tuli ongelmia aiemmin käytetyn float-left (teksti) ja float-right (nappi) kanssa, alkoivat mennä päällekkäin. Nykyratkaisussa ne piirretään flowssa inline-blockina, joten pysyvät samalla rivillä vierekkäin jos tilaa, muuten rivittyy nätisti menemättä päällekkäin.